### ISSUE TITLE
Problems loading /details/timezone from backend, please reload

### ISSUE THREAD

**Issue by m3th05** [mrworf/photoframe#59]
Hi

I keep on getting the following error on the config page.

Also, no providers is my list of providers to pick from.

![capture](https://user-images.githubusercontent.com/6140156/51596049-c1006380-1f00-11e9-8574-442755a982f8.PNG)
![capture2](https://user-images.githubusercontent.com/6140156/51596509-d0cc7780-1f01-11e9-94ab-112cc9000331.PNG)

**Comment by mrworf** [comment]
Are you using the SD Card image or did you install manually? Sounds like you're missing the `/usr/bin/timedatectl` ... try running it locally and see if it exists.

**Comment by m3th05** [comment]
> Are you using the SD Card image or did you install manually? Sounds like you're missing the `/usr/bin/timedatectl` ... try running it locally and see if it exists.

I have manually installed it, It was version 1 something when I updated it.
I have been having these issues since version 2.

I see all the timezone when i run the above command, so that is not the issue. It seems like it has something to do with the json format. When I validate the json format online, it is invalid: ['Africa/Johannesburg', 'Africa/Harare']

I have the same issue with the provider's list,

I will be doing a fresh install from the img today.
